modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/utils/RestUtils.java

```diff
@@ -123,6 +124,9 @@
             externalIds.put(SW360Attributes.RELEASE_EXTERNAL_ID_COPYRIGHTS, sw360Release.getCopyrights());
         }
 
+        if(externalIds.isEmpty()) {
+            return null;
```

Comment:
why should `null` be better than a empty map? This is just prone to NPEs and requests for more null-checks later